data/models/Company.js

```diff
@@ -62,6 +62,13 @@ module.exports = (sequelize, DataTypes) => {
             as: 'companyRoles',
             foreignKey: 'companyId'
         });
+
+        Company.belongsToMany(models.TimesheetRecord, {
+            as: 'timesheetRecords',
+            through: 'CompanyRole',
```

Comment:
Wrong type of association
Company has many timesheet records
Timesheet record belongs to a company

Reply:
fix